std/experimental/checkedint.d

```diff
@@ -292,9 +292,24 @@ if (isIntegral!T || is(T == Checked!(U, H), U, H))
             is(typeof(Checked!(T, Hook)(rhs.get))))
     {
         static if (isIntegral!U)
+        {
```

Comment:
Please do only the necessary changes. Adding brackets here does not add any value to this PR